Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/SetPoints/Subsequences/PartLoadRatios.mo

```diff
@@ -44,545 +44,571 @@ block PartLoadRatios
+    annotation (Placement(transformation(extent={{-200,260},{-180,280}})));
 
   Buildings.Controls.OBC.CDL.Integers.Max maxIntUp "Maximum"
-    annotation (Placement(transformation(extent={{-240,160},{-220,180}})));
+    annotation (Placement(transformation(extent={{-200,160},{-180,180}})));
 
   Buildings.Controls.OBC.CDL.Integers.Max maxIntDown "Maximum"
-    annotation (Placement(transformation(extent={{-240,100},{-220,120}})));
+    annotation (Placement(transformation(extent={{-200,100},{-180,120}})));
 
   Buildings.Controls.OBC.CDL.Continuous.Max max if anyVsdCen "Maximum"
-    annotation (Placement(transformation(extent={{180,-420},{200,-400}})));
+    annotation (Placement(transformation(extent={{220,-420},{240,-400}})));
 
   Buildings.Controls.OBC.CDL.Continuous.Min min if anyVsdCen "Minimum"
-    annotation (Placement(transformation(extent={{220,-380},{240,-360}})));
+    annotation (Placement(transformation(extent={{260,-380},{280,-360}})));
 
   Buildings.Controls.OBC.CDL.Continuous.Sources.Constant maxLim(
     final k=varSpeStaMax) if anyVsdCen "Constant"
-    annotation (Placement(transformation(extent={{120,-360},{140,-340}})));
+    annotation (Placement(transformation(extent={{160,-360},{180,-340}})));
 
   Buildings.Controls.OBC.CDL.Continuous.Sources.Constant minLim(
     final k=varSpeStaMin) if anyVsdCen "Constant"
-    annotation (Placement(transformation(extent={{120,-420},{140,-400}})));
+    annotation (Placement(transformation(extent={{160,-420},{180,-400}})));
 
   Buildings.Controls.OBC.CDL.Utilities.Assert cheStaTyp1(
     final message="Recommended staging order got violated or an unlisted chiller type got provided when staging down")
     "Chiller type outside of recommenation when staging down"
-    annotation (Placement(transformation(extent={{220,-20},{240,0}})));
+    annotation (Placement(transformation(extent={{260,-20},{280,0}})));
 
   Buildings.Controls.OBC.CDL.Continuous.Sources.Constant conSpeCenTypMult1(
     final k=anyOutOfScoMult)
     "Outside of G36 recommended staging order chiller type SPLR multiplier"
-    annotation (Placement(transformation(extent={{160,20},{180,40}})));
+    annotation (Placement(transformation(extent={{200,20},{220,40}})));
 
   Buildings.Controls.OBC.CDL.Logical.Switch swi5
     "Logical switch"
-    annotation (Placement(transformation(extent={{280,220},{300,240}})));
+    annotation (Placement(transformation(extent={{320,220},{340,240}})));
 
   Buildings.Controls.OBC.CDL.Logical.Switch swi6
     "Logical switch"
-    annotation (Placement(transformation(extent={{220,-110},{240,-90}})));
+    annotation (Placement(transformation(extent={{260,-110},{280,-90}})));
+
+  Buildings.Controls.OBC.CDL.Continuous.Sources.Constant smaNum(
+    final k=small) "Small constant"
+    annotation (Placement(transformation(extent={{-360,20},{-340,40}})));
+
+  Buildings.Controls.OBC.CDL.Continuous.Max max1
+    "Maximum"
```

Comment:
May modify the comment to something like `Maximum output to avoid zero denominator in downstream`.